### PR TITLE
Fixed: Added contributors section to README.md

### LINKED ISSUE #1184: [Docs]:
### Where is the problem?

No contributors section to the README.md file.

### What can we do better?

There is no Contributors section in readme file .
As we know Contributions are what make the open-source community such an amazing place to learn, inspire, and create.
The Contributors section in a README.md file is important as it acknowledges and gives credit to those who have contributed to a project, fosters community and collaboration, adds transparency and accountability, and helps document the project's history for current and future maintainers. It also serves as a form of recognition, motivating contributors to continue their efforts.
contributors

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by Kalyanimhala**
Fixed #1184 
Added Contributors section to Readme.
@lattner  @jackos kindly view it and merged the PR. Thank You!

### CODE REVIEW

**Comment by Kalyanimhala** [comment]
@jackos Please view my PR!

File: README.md [hunk 1]
```diff
@@ -25,3 +25,8 @@ Otherwise, you can:
 - Read the [inspiration behind Mojo](https://docs.modular.com/mojo/why-mojo.html).
 - Check out the [Mojo programming manual](https://docs.modular.com/mojo/programming-manual.html).
 - Read our other docs on [docs.modular.com/mojo](https://docs.modular.com/mojo).
+
+### All Thanks To Our Contributors:
+<a href="https://github.com/jackos/mojo/graphs/contributors">
+  <img src="https://contrib.rocks/image?repo=jackos/mojo" />
```
Comment (jackos) [inline README.md:31]: Repo should be `modularml/mojo` not `jackos/mojo`

File: README.md [hunk 2]
```diff
@@ -25,3 +25,8 @@ Otherwise, you can:
 - Read the [inspiration behind Mojo](https://docs.modular.com/mojo/why-mojo.html).
 - Check out the [Mojo programming manual](https://docs.modular.com/mojo/programming-manual.html).
 - Read our other docs on [docs.modular.com/mojo](https://docs.modular.com/mojo).
+
+### All Thanks To Our Contributors:
```
Comment (jackos) [inline README.md:29]: Can you please change to `## Thanks To Our Contributors`

`##` will look better

File: README.md [hunk 3]
```diff
@@ -25,3 +25,8 @@ Otherwise, you can:
...
+<a href="https://github.com/jackos/mojo/graphs/contributors">
```
Comment (jackos) [inline README.md:30]: Change to:

`https://github.com/modularml/mojo/graphs/contributors`

**Review by jackos — CHANGES_REQUESTED** [review]
Looks great thanks, if you can please make the requested changes

**Review by jackos — APPROVED** [review]
Great thanks!

**Pull request merged**